sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -995,6 +995,7 @@ public long count(ContactCriteria contactCriteria) {
 		}
 
 		cq.select(cb.count(root));
+		cq.distinct(true);
```

Comment:
I don't think that's gonna work; please check CaseFacadeEjb.count, you should do a countDistinct instead